File: PRAK603-2310817220016-NailaHanifah.c
#include <stdio.h>

void multiplyRows(int n, int row1[n], int row2[n], int result[n]) {
    for (int i = 0; i < n; ++i) {
        result[i] = row1[i] * row2[i];
    }
}

int main() {
    int n1, n2;
    scanf("%d %d", &n1, &n2);

    if (n1 != n2) {
        printf("Jumlah tidak sama\n");
    } else {
        int data[2 * n1];
        for (int i = 0; i < n1; ++i) {
            scanf("%d", &data[i]);
        }
        for (int i = n1; i < 2 * n1; ++i) {
            scanf("%d", &data[i]);
        }

        int result[n1];
        multiplyRows(n1, data, data + n1, result);

        for (int i = 0; i < n1; ++i) {
            printf("%d ", result[i]);
        }
        printf("\n");
    }

    return 0;
}

File: PRAK603-2310817220016-NailaHanifah.py
def multiply_rows(n, row1, row2):
    result = [row1[i] * row2[i] for i in range(n)]
    return result

def main():
    n1, n2 = map(int, input().split())

    if n1 != n2:
        print("Jumlah tidak sama")
    else:
        data = []

        row1 = list(map(int, input().split()))
        data.extend(row1 * 2)

        row2 = list(map(int, input().split()))
        data.extend(row2 * 2)

        result = multiply_rows(n1, row1, row2)

        print(" ".join(map(str, result)))

if __name__ == "__main__":
    main()